src/mqt/predictor/rl/actions/bqskit_actions.py

```
@@ -81,54 +102,34 @@ def bqskit_to_qiskit(circuit: Circuit) -> QuantumCircuit:
-        CompilationOrigin.BQSKIT,
-        PassType.SYNTHESIS,
-        transpile_pass=lambda device: (
-            lambda bqskit_circuit: bqskit_compile(
-                bqskit_circuit,
-                model=MachineModel(bqskit_circuit.num_qudits, gate_set=get_bqskit_native_gates(device)),
-                **_bqskit_compilation_options(),
-            )
-        ),
-    )
+def _run_bqskit_workflow(
+    circuit: Circuit, workflow: Workflow, request_data: bool = False
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)